patches/content-renderer-render_frame_impl.cc.patch

```diff
@@ -0,0 +1,23 @@
+diff --git a/content/renderer/render_frame_impl.cc b/content/renderer/render_frame_impl.cc
+index c110db6a32bba72c7a89137fa39377c58f0061d8..608696dbd137ee57075a80528ca8a3f5611b94e8 100644
+--- a/content/renderer/render_frame_impl.cc
++++ b/content/renderer/render_frame_impl.cc
+@@ -47,6 +47,8 @@
+ #include "base/threading/thread_task_runner_handle.h"
+ #include "base/time/time.h"
+ #include "base/trace_event/trace_event.h"
++#include "brave/common/brave_features.h"
++#include "brave/common/network_constants.h"
+ #include "build/build_config.h"
+ #include "cc/base/switches.h"
+ #include "content/common/associated_interfaces.mojom.h"
+@@ -4788,6 +4790,9 @@ void RenderFrameImpl::WillSendRequestInternal(
+     request.SetHttpHeaderField(blink::WebString::FromUTF8(kDoNotTrackHeader),
+                                "1");
+   }
++  if (base::FeatureList::IsEnabled(features::kGlobalPrivacyControl)) {
```

Comment:
same as https://github.com/brave/brave-core/pull/6743/files#r498456726 - but also for future reference please see the patching guidelines because these patches aren't in the correct format https://github.com/brave/brave-browser/wiki/Patching-Chromium